fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/observer/EventType.java

```diff
@@ -15,25 +15,27 @@
  */
 package org.fcrepo.kernel.api.observer;
 
+import static org.fcrepo.kernel.api.RdfLexicon.EVENT_NAMESPACE;
+
 /**
- * A convenient abstraction over JCR's integer-typed events.
+ * A collection of repository event types
  *
  * @author ajs6f
  * @since Feb 7, 2013
  */
 public enum EventType {
-    NODE_ADDED("node added"),
-    NODE_REMOVED("node removed"),
-    PROPERTY_ADDED("property added"),
-    PROPERTY_REMOVED("property removed"),
-    PROPERTY_CHANGED("property changed"),
-    NODE_MOVED("node moved"),
-    PERSIST("persist");
+
+    RESOURCE_CREATION("resource creation", "ResourceCreation"),
```

Comment:
COPY is interpreted the same as it always has been: as the creation of a new resource, i.e. RESOURCE_CREATION or what used to be NODE_ADDED.